scripts/build_index.rb

```diff
@@ -20,6 +20,6 @@
     end
 end
 
-commands = commands.map do |k,v| v end
+commands = commands.sort.map do |k,v| v end
```

Comment:
No need for that. Take a look at the index.json. That the result after I modify the script.